session.go

```diff
@@ -112,6 +113,7 @@ func newSession(config *Config, conn net.Conn, client bool, readBuf int) *Sessio
 		acceptCh:   make(chan *Stream, config.AcceptBacklog),
 		sendCh:     make(chan []byte, 64),
 		pongCh:     make(chan uint32, config.PingBacklog),
+		pingCh:     make(chan uint32),
```

Comment:
should this be `close()`'d in the Close() method?

Reply:
No. Neither the pong nor the ping channel should ever be closed (could cause a panic).